level1c4pps/eumgacfdr2pps_lib.py

```diff
@@ -229,6 +256,12 @@ def process_one_file(eumgacfdr_file, out_path='.', reader_kwargs=None, engine='h
     # Handle gac specific datasets qual_flags and scanline_timestamps
     update_ancilliary_datasets(scn_)
 
+    if start_line is not None and end_line is not None:
+        # Crop after all renaming of variables are done
+        # Problems to rename if cropping is done first.
```

Comment:
Which problems exactly did you encounter?

Reply:
This (in function rename_latitude_longitude in file __init__.py) no longer works:
 scene['lat'] = scene[lat_name_satpy]

Reply:
Hmm, that's strange.